src/internal/connector/graph/service_helper.go

```diff
@@ -51,12 +50,7 @@ func CreateAdapter(tenant, client, secret string) (*msgraphsdk.GraphRequestAdapt
 func CreateHTTPClient() *nethttp.Client {
 	clientOptions := msgraphsdk.GetDefaultClientOptions()
 	middlewares := msgraphgocore.GetDefaultMiddlewaresWithOptions(&clientOptions)
-
-	// When true, additional logging middleware support added for http request
-	if os.Getenv(logGraphRequestsEnvKey) != "" {
-		middlewares = append(middlewares, &LoggingMiddleware{})
-	}
-
+	middlewares = append(middlewares, &LoggingMiddleware{})
```

Comment:
I like the idea behind this, but before we enable it everywhere should we check if it has any large impacts on memory usage, runtime, etc?

Reply:
This is a standard pattern, so I'm not particularly concerned about performance impacts.  Though I recognize that I kept the request body dump around from prior debugging.  We can ditch that for now and re-add if necessary in the future.